using-nats/developing-with-nats/services/README.md

```diff
@@ -0,0 +1,23 @@
+# Building NATS Services
+Recently we have agreed upon an [initial specification](https://github.com/nats-io/nats-architecture-and-design/blob/main/adr/ADR-32.md) for a services protocol so that we can add first-class services support to NATS clients and support this in our tooling. This services protocol is an agreement between clients and tooling and doesn't require any special functionality from the NATS server or JetStream.
+
+To check if the NATS client in your favorite language supports the new services API, make sure you check the docs and GitHub repository for that client. The services API is relatively new and not all clients may support it yet.
```

Comment:
You could also add a link to NBE and refer people to the "Micro Framework" (which we should really rename to "Services").